scripts/sign_rproc_fw.py

```diff
@@ -0,0 +1,509 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: BSD-2-Clause
+#
+# Copyright (C) 2023, STMicroelectronics - All Rights Reserved
+#
+
+try:
+    from elftools.elf.elffile import ELFFile
+    from elftools.elf.sections import SymbolTableSection
+    from elftools.elf.enums import ENUM_P_TYPE_ARM
+    from elftools.elf.enums import *
+except ImportError:
+    print("""
+***
+ERROR: "pyelftools" python module is not installed or version < 0.25.
+***
+""")
+    raise
+
+try:
+    from Cryptodome.Hash import SHA256
+    from Cryptodome.Signature import pkcs1_15
+    from Cryptodome.PublicKey import RSA
+    from Cryptodome.Signature import DSS
+    from Cryptodome.PublicKey import ECC
+except ImportError:
+    print("""
+            ***
+            ERROR: "pycryptodomex" python module should be installed.
+            ***
+        """)
+    raise
+
+import os
+import sys
+import struct
+import logging
+import binascii
+
+#  Generated file structure:
+#
+#                   -----+-------------+
+#                  /     |    Magic    |  32-bit word, magic value equal to
+#                 /      +-------------+  0x3543A468
+#                /       +-------------+
+#               /        |   version   |  32-bit word, version of the format
+#              /         +-------------+
+# +-----------+          +-------------+
+# |   Header  |          |  sign size  |  32-bit word, size of the signature
+# +-----------+          +-------------+  (aligned on 64-bit), in bytes.
+#              \         +-------------+
+#               \        | images size |  32-bit word, size of the images to
+#                \       +-------------+  load (aligned on 64-bit), in bytes.
+#                 \      +-------------+
+#                  \     |  TLV size   |  32-bit word, size of the generic TLV
+#                   -----+-------------+  (aligned on 64-bit), in bytes.
+#
+#                        +-------------+  Information used to authenticate the
+#                        |     TLV     |  images and boot the remote processor,
+#                        |             |  stored in Type-Length-Value format.
+#                        +-------------+  'Type' and 'Length' are 32-bit words.
+#
+#                        +-------------+
+#                        | Signature   |   Signature of the header, the TLV,
+#                        +-------------+   and the platform TLV.
```

Comment:
I guess we can skip "and the platform TLV" since that's now part of the TLV.

Reply:
Right, I 'm waiting @etienne-lms feedback and I will fix this.

Reply:
I agree with Jens, TLV covers cells for TA and PTA. Can remove "and the platform TLV" here.